deploy.bat

```diff
@@ -5,6 +5,10 @@ SET API_KEY=%2
 SET ROOT_DIR=%~dp0 
 cd %ROOT_DIR%
 
-dotnet pack Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n --output %ROOT_DIR%
+dotnet build Snowflake.Data\Snowflake.Data.csproj -c Release --force -v n 
+
+REM command to sign with strong name Snowflake.Data.dll should be here

```

Comment:
@sfc-gh-knozderko The deploy.bat file was updated to be prepare for the sign. Between the dotnet build and dotnet pack the assembly should be sign. Otherwise, the DLL could be distributed without the strong name.